README.md

```diff
@@ -0,0 +1,119 @@
+
+  Aquae - gov.uk Personal Data Exchange Specification
+  ===================================================
+
+  Copyright (C) 2017, The Personal Data Exchange Team, Crown Copyright (Government Digital Service).
+
+## 1. Introduction
+
+gov.uk Personal Data Exchange is a way to query existing personal data
+held by government. This distribution is the technical specification
+for how the system operates and a set of tools for working with the
+messages and protocols defined.
+
+There are a number of files supplied:
```

Comment:
The file list is already out-of-date... should we just remove this? We could replace it with a general description of what is in each folder, or do a per-file type thing.

Reply:
Agree. I shuffled things around a bit and kept just the specification files and added a section about the examples directory. Better?